rflib/intelhex.py

```diff
@@ -343,15 +358,15 @@ def addresses(self):
         '''Returns all used addresses in sorted order.
         @return         list of occupied data addresses in sorted order. 
         '''
-        aa = self._buf.keys()
+        aa = list(self._buf.keys())
         aa.sort()
         return aa
 
     def minaddr(self):
         '''Get minimal address of HEX content.
         @return         minimal address or None if no data
         '''
-        aa = self._buf.keys()
+        aa = list(self._buf.keys())
```

Comment:
i notice a lot of this.  is there a problem accessing the list returned from keys() in Py3?  or is this a stylistic thing?

Reply:
In Python 3 the `dict.keys()` will return a `dict_keys` object instead of a list, e.g.:
```python
>>> my_dict = {'first': 1, 'second': 2} 
>>> my_dict.keys()
dict_keys(['first', 'second'])
>>> list(my_dict.keys())
['first', 'second']
```

For more information see: [PEP 3106 -- Revamping dict.keys(), .values() and .items()](https://www.python.org/dev/peps/pep-3106/).